origin-discovery/devops/es-cli.js

```diff
@@ -1,16 +1,35 @@
 #!/usr/bin/env node
 
-const querystring = require('querystring')
+/**
+ * CLI for managing Elastic Search indices.
+ *
+ */
+
+require('dotenv').config()
+
+try {
+  require('envkey')
+} catch (error) {
+  console.log('EnvKey not configured')
+}
+
 const http = require('http')
 const https = require('https')
 const readline = require('readline')
-const arrayFunctions = require('./../../../origin-js/src/utils/arrayFunctions')
+const { groupBy, mapValues } = require('./../../origin-js/src/utils/arrayFunctions.js')
 
 // Configuration
-const host = 'localhost'
-const port = '9200'
+var host = 'localhost'
```

Comment:
I think the linter is going to complain about `var` here. We have a universal linter setup now, you can run `npm run lint` from root or `lerna run lint --scope origin-discovery` for just linting discovery.

I'm sure you'll lern this. 😆

Reply:
oh sorry. just saw this.
ah looks like linter did not catch me because that directory is not included ! :)
I'll fix the linter config and that file. Thanks for catching this - Mr Linter ;)